### DISCUSSION TITLE
Order binding

### DISCUSSION THREAD

**Discussion by Settleman** [Windos/BurntToast#240] [Q&A]
Hi all, I just found this amazing component. thanks for all the work put in.

My question is if it's possible to order the information when showing the notification.

```
$Binding1 = New-BTBinding -Column $Column1, $Column2 -Children $text, $logo -AppLogoOverride $icon
$Visual1 = New-BTVisual -BindingGeneric $Binding1
$Content1 = New-BTContent -Visual $Visual1 -Header $title -Actions $bta
Submit-BTNotification -Content $Content1

```
For instance, I want to show  $title on top, $logo next, then $text with $icon and finally the columns.
![image](https://github.com/user-attachments/assets/fba436d1-2507-4f6d-b2c0-c54e8e11cf4a)

Thanks in advance.

Cheers
C

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
